app.rb

```diff
@@ -0,0 +1,68 @@
+# app.rb
```

Comment:
![image](https://github.com/DevKaranJ/school-library/assets/93039359/e9f62a91-0495-4506-a19e-84d611005b9b)

- [ ] It would be nice if you fixed your link because when I tried the LinkedIn link it redirected me to your X account 😨 Kindly add the right link to make your readme descriptive and professional 👍